Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```
@@ -583,6 +583,9 @@
   - type: Tag
     tags:
     - Kangaroo
+  - type: PhysicalComposition
+    materialcomposition: 
```

needs to be camelcased
```suggestion
    materialComposition: 
```